stl/inc/xtree

```diff
@@ -1321,14 +1321,27 @@ public:
     }
 
     size_type erase(const key_type& _Keyval) noexcept(noexcept(_Eqrange(_Keyval))) /* strengthened */ {
-        const auto _Where = _Eqrange(_Keyval);
+        return _Erase(_Eqrange(_Keyval));
+    }
+
+private:
+    size_type _Erase(const pair<_Nodeptr, _Nodeptr> _Where) noexcept {

```

Comment:
Not a big fan of putting it here. Can we move that to the other helpers at the end

Reply:
I wanted to make a smaller diff, but maybe you are right

Reply:
Nope, there's no place for other helpers, private and public blocks already alternate for other methods